.github/workflows/release_actions.yml

```
@@ -0,0 +1,53 @@
+  push:
+    branches:
+      - main
+
+jobs:
+  publish-docker-image:
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)